parcel.go

```diff
@@ -13,48 +13,82 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	//добавление строки в таблицу parcel, используйте данные из переменной p
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :addres, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("addres", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, nil
+	}
+
+	//идентификатор последней добавленной записи
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, nil
```

Comment:
Нужно вернуть ошибку или какой-то внятный текст
```suggestion
		return 0, err
```